### ISSUE TITLE
"Time dependence" plot x-axis looks wrong

### ISSUE THREAD

**Issue by pberkes** [wichmann-lab/python-psignifit#195]
Looking at the demo `plotting.md`, I noticed that in the `plot_modelfit` plot the 2nd and 3rd plots are identical. The 3rd plot seems wrong: the x-axis label says "block number", but the x-values are the stimulus values.

<img width="1175" alt="image" src="https://github.com/user-attachments/assets/68464e14-bf63-4b35-94bb-6dff7d3adbfd" />

**Comment by guillermoaguilar** [comment]
Yeah it should be 1, 2, 3... . It is assumed that the rows in the data come in the same order as it was presented in the experiment, each row contains the results of a block of trials with a particular value of the stimulus. 
(see here from the old version of psignifit https://psignifit.sourceforge.net/QUICKSTART.html#experimental-scenario-and-data-format)

I'm thinking that maybe it would be nice to have an example where the stimulus values and the presentation order was not in the same sequence, so one sees the difference between the middle and right plot...

I'll take a look
